Extending/Key-Vault/index.md

```diff
@@ -0,0 +1,79 @@
+---
+versionFrom: 9.0.0
+state: complete
+meta.Title: "Azure Key Vault"
+meta.Description: "A guide configuring an Azure Key Vault"
+---
+
+# Configuring an Azure Key Vault
+
+From a security perspective, it's always a good solution to store your application secrets such as a connection string and others keys in an Azure Key Vault. This article tells you how to configure your application so that it's ready to use a Key Vault
```

Comment:
```suggestion
From a security perspective, it's always a good solution to store your application secrets (such as a connection string and others keys) in an Azure Key Vault. This article tells you how to configure your application so that it's ready to use a Key Vault
```